src/commands/up.rs

```diff
@@ -458,6 +461,17 @@ impl AppSource {
     }
 }
 
+impl Display for AppSource {
```

Comment:
This reads more like a Debug implementation than a Display one.

Reply:
This is responsible for the updated error message here: https://github.com/fermyon/spin/pull/1780/files#diff-82f776c9616e3a5ecda575278ca8de8a920780afe36b0fff8ab7252292425aad